src/main/webapp/readfile.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Read File Configuration</title>
+        <link rel="stylesheet" href="readfile.css">
+        <script src="/scripts/script.js"></script>
+    </head>
+    <body id="body">
+        <h1>Billing Configuration Sync Tool</h1>
+        </br>
+        <h2>Read Configuration</h2> 
+        <!-- Links Div -->
+        <div>
+        </div>
```

Comment:
I think it would be clearer to give the div an ID that indicates this is where the Links Div goes, rather than a comment